erpnext/manufacturing/report/production_order_stock_report/production_order_stock_report.py

```diff
@@ -0,0 +1,132 @@
+# Copyright (c) 2017, Velometro Mobility Inc and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+from frappe.utils import flt, cint
+import frappe
+
+def execute(filters=None):
+	
+	
+	prod_list = get_production_orders()
+	
+	data = get_item_list( prod_list)
+	
+	columns = get_columns()
+	
+	return columns, data
+
+	
+def get_item_list(prod_list):
+	
+	out = []
+	
+	low_price_data = []
+	low_supplier = []
+	company = frappe.db.get_default("company")
+	float_precision = cint(frappe.db.get_default("float_precision")) or 2
+	company_currency = frappe.db.get_default("currency")
+	# Get the default supplier of suppliers
```

Comment:
What is the use of company, float_precision, company_currency this never used in the code